ClaudiaHernandezGarcia/springBootProject/src/main/java/com/globant/springBootProject/service/AnagramsService.java

```diff
@@ -0,0 +1,7 @@
+package com.globant.springBootProject.service;
+
+import com.globant.springBootProject.dto.AnagramsDTO;
+
+public interface AnagramsService {
+    AnagramsDTO Solution(AnagramsDTO anagramsDTO);
```

Comment:
Solution??? uppercase for a method name??
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names